src/lib/types.hpp

```diff
@@ -219,7 +219,7 @@ enum class CleanupTemporaries : bool { Yes = true, No = false };
 // Used as a template parameter that is passed whenever we conditionally erase the type of a template. This is done to
 // reduce the compile time at the cost of the runtime performance. Examples are iterators, which are replaced by
 // AnySegmentIterators that use virtual method calls.
-enum class EraseTypes { OnlyInDebug, Always };
+enum class EraseTypes { OnlyInDebugBuild, Always };
```

Comment:
For me, this reads nicer in code, but will revert without discussion if anyone thinks differently. :speak_no_evil: